tests/components/renault/test_config_flow.py

```diff
@@ -176,7 +176,11 @@ async def test_config_flow_duplicate(hass: HomeAssistant):
         "homeassistant.components.renault.async_setup_entry",
         return_value=True,
     ) as mock_setup_entry:
-        await setup_renault_integration_simple(hass)
+        config_entry = get_mock_config_entry()
+        config_entry.add_to_hass(hass)
+        await hass.config_entries.async_setup(config_entry.entry_id)
```

Comment:
We don't need to set up the integration to test the duplicate config entry handling.

Reply:
@MartinHjelmare sorry for the delay - it should be ok now.